<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="2.0">
<context>
    <name>@default</name>
    <message>
        <location filename="../knossos/ipc.py" line="167"/>
        <source>ipc</source>
        <comment>Mod &quot;%s&quot; could not be found!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/ipc.py" line="171"/>
        <source>ipc</source>
        <comment>The mod &quot;%s&quot; could not be found due to an internal error!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/ipc.py" line="191"/>
        <source>ipc.handle_message</source>
        <comment>The fso://open/&lt;mod id&gt; link is missing a parameter!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/ipc.py" line="202"/>
        <source>ipc.handle_message</source>
        <comment>The mod &quot;%s&quot; was not found!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/ipc.py" line="205"/>
        <source>ipc.handle_message</source>
        <comment>The action &quot;%s&quot; is unknown!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="424"/>
        <source>SettingsWindow</source>
        <comment>Failed to save the configuration!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="646"/>
        <source>SettingsWindow</source>
        <comment>An unknown error (a crash?) occurred while trying to load the libraries!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="650"/>
        <source>SettingsWindow</source>
        <comment>Success! Both libraries were loaded successfully.</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="652"/>
        <source>SettingsWindow</source>
        <comment>Error! One or both libraries failed to load.</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="654"/>
        <source>SettingsWindow</source>
        <comment>Not found</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="687"/>
        <source>SettingsWindow</source>
        <comment>The FSO binary &quot;%s&quot; is missing!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="692"/>
        <source>SettingsWindow</source>
        <comment>The FSO binary &quot;%s&quot; is missing %s!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="694"/>
        <source>SettingsWindow</source>
        <comment>The FSO binary quit with code %d!</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/settings.py" line="736"/>
        <source>SettingsWindow</source>
        <comment>Sorry, but I can&apos;t find the fs2_open.log file.
Did you run the debug build?</comment>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>GOGExtractTask</name>
    <message>
        <location filename="../knossos/tasks.py" line="1878"/>
        <source>The selected file wasn&apos;t a proper Inno Setup installer. Are you sure you selected the right file?</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>HellWindow</name>
    <message>
        <location filename="../knossos/windows.py" line="366"/>
        <source>There&apos;s an update available!
Do you want to install Knossos %s now?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/windows.py" line="375"/>
        <source>There&apos;s an update available!
You should update to Knossos %s.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>InstallTask</name>
    <message>
        <location filename="../knossos/tasks.py" line="416"/>
        <source>The mod installation was aborted before it could finish. Uninstall the partially installed mod %s or verify the file integrity.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/tasks.py" line="420"/>
        <source>An error occured during the installation of %s. It might be partially installed.
</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>MacUpdateTask</name>
    <message>
        <location filename="../knossos/tasks.py" line="2086"/>
        <source>Failed to launch the update!</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ModInstallWindow</name>
    <message>
        <location filename="../knossos/windows.py" line="682"/>
        <source>I&apos;m sorry but you won&apos;t be able to install &quot;%s&quot; because &quot;%s&quot; is missing!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/windows.py" line="729"/>
        <source>Installed</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>WebBridge</name>
    <message>
        <location filename="../knossos/web.py" line="306"/>
        <source>I can&apos;t uninstall this mod because it&apos;s in dev mode!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="331"/>
        <source>You can&apos;t uninstall this because %s still depend on it.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="337"/>
        <source>Do you really want to uninstall %s?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="340"/>
        <source>%s will be removed.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="516"/>
        <source>The selected path is not to root_fs2.vp!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="521"/>
        <source>The selected root_fs2.vp&apos;s folder does not have the FreeSpace 2 files!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="542"/>
        <source>The selected path is not a directory!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="546"/>
        <source>Using a folder in &quot;Program Files&quot; for the Knossos library is not recommended, because you will have to always run Knossos as Administrator. Use anyway?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="557"/>
        <source>Using a non-empty folder for the Knossos library is not recommended, because it can cause problems for Knossos. Use anyway?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="564"/>
        <source>The selected path does not exist. Should I create the folder?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="571"/>
        <source>Failed to create Knossos data folder!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="578"/>
        <source>Please don&apos;t use an existing FS2 directory. It won&apos;t work! Select an empty directory instead.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="720"/>
        <source>The selected path is not a directory and not an installer!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="723"/>
        <source>The selected path does not contain the retail files!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="750"/>
        <source>The selected parent TC is not valid!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="768"/>
        <source>There already exists a mod with the chosen ID!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="782"/>
        <source>Knossos couldn&apos;t check if your mod ID is unique because it couldn&apos;t connect to the Nebula. Continue at your own risk if you&apos;re sure it is unique, otherwise please abort.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="787"/>
        <source>Your chosen mod ID is already being used by someone else. Please choose a different one.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="798"/>
        <source>I could not create the folder for binaries!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="805"/>
        <source>I could not create the folder for the TC!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="809"/>
        <source>The chosen parent does not exist! Something went very wrong here!!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="820"/>
        <source>Ini mods with a primary list are currently not supported for importing!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="846"/>
        <source>The mod.ini dependency %s is not known to Knossos and could not be converted to a mod.json dependency.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1586"/>
        <source>Failed to save mod.json!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1528"/>
        <source>You can&apos;t edit &quot;%s&quot; because it isn&apos;t in dev mode!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="955"/>
        <source>Please select an image</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="955"/>
        <source>Image (*.png *.jpg *.jpeg *.gif *.bmp)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="968"/>
        <source>Executables (*.exe)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="972"/>
        <source>Please select one or more executables</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1101"/>
        <source>Internal data inconsistency. Please try again.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1246"/>
        <source>Failed to find the mod! Weird...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1076"/>
        <source>The entered first release date is invalid!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1083"/>
        <source>The entered last update date is invalid!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1123"/>
        <source>Failed to find the package! Weird...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1208"/>
        <source>Failed to save the selected FSO build. Make sure that you have at least one required package!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1271"/>
        <source>Failed to save user.json!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1569"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1523"/>
        <source>Somehow I lost the mod you&apos;re talking about! I&apos;m sorry, this is a bug.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1536"/>
        <source>The specified version number is invalid!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1541"/>
        <source>Failed to parse the new version! This is a bug.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1547"/>
        <source>The new version has to be higher than the old version!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1556"/>
        <source>The destination folder (%s) already exists! I won&apos;t overwrite an existing folder!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/web.py" line="1569"/>
        <source>Failed to rename folder &quot;%s&quot;! Make sure that no other program has locked it.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>WindowsUpdateTask</name>
    <message>
        <location filename="../knossos/tasks.py" line="2050"/>
        <source>Failed to launch the update!</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>launcher</name>
    <message>
        <location filename="../knossos/launcher.py" line="86"/>
        <source>A critical error occurred! Knossos might not work correctly until you restart it.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/launcher.py" line="88"/>
        <source>The error has been reported and will hopefully be fixed soon.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/launcher.py" line="90"/>
        <source>If you want to help, report this bug on our Discord channel, in the HLP thread or on GitHub. Just click a button below to open the relevant page.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/launcher.py" line="197"/>
        <source>I can&apos;t find &quot;7z&quot;! Please install it and run this program again.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/launcher.py" line="248"/>
        <source>I don&apos;t know how to handle &quot;%s&quot;! I only know fso:// .</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/launcher.py" line="256"/>
        <source>Not enough arguments!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/launcher.py" line="269"/>
        <source>Failed to start server!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/launcher.py" line="396"/>
        <source>I encountered a fatal error.
I&apos;m sorry but I&apos;m going to crash now...</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>runner</name>
    <message>
        <location filename="../knossos/runner.py" line="229"/>
        <source>Starting %s failed! (%s)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="230"/>
        <source>Failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="234"/>
        <source>I can&apos;t find the selected executable! (The file &quot;%s&quot; is missing!)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="240"/>
        <source>I can&apos;t start because the libraries %s are missing!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="242"/>
        <source>I can&apos;t start because the library %s is missing!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="248"/>
        <source>Your fs2_open.ini contains a joystick which this FSO version can&apos;t detect and Knossos doesn&apos;t remember your joystick. Please go to Knossos&apos; settings, select your joystick, save and try again.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="435"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="407"/>
        <source>The mod &quot;%s&quot; could not be found!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="419"/>
        <source>I couldn&apos;t find the tool&apos;s executables! Aborted.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="435"/>
        <source>I couldn&apos;t find an executable. Aborted.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../knossos/runner.py" line="442"/>
        <source>No matching executable was found!</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>tasks</name>
    <message>
        <location filename="../knossos/tasks.py" line="1878"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>util.human_list</name>
    <message>
        <location filename="../knossos/util.py" line="737"/>
        <source> and </source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
